README.md

```diff
@@ -20,8 +20,8 @@
   <a href="https://opensource.org/licenses/Apache-2.0">
     <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="Foton is licensed with Apache License 2.0">
   </a>
-  <a href="https://www.repostatus.org/#active">
-  <img src="https://www.repostatus.org/badges/latest/active.svg" alt="Project Status: Active – The project has reached a stable, usable state and is being actively developed."/>
+  <a href="https://www.repostatus.org/#experimental">
+  <img src="https://github.com/GIScience/badges/raw/master/status/experimental.svg" alt="Project Status: Experimental – The project has reached a stable, usable state and is being actively developed."/>
```

Comment:
it has not yet reached a stable state (we may need to change the API without backwards compatibility)